### DISCUSSION TITLE
User and ACL management only?

### DISCUSSION THREAD

**Discussion by mnarrell** [lae/ansible-role-proxmox#182] [Q&A] [answered]
Can I use this module to only manage users/group/roles/etc, and not provision a cluster/standalone?

**Comment by lae** [comment, marked as answer]
This role is for primarily for deploying Proxmox. If you're going to use the role as-is, using it in standalone (clustering disabled) would be a minimum requirement.

It would be possible for you to extract the `proxmox_user` and other modules and use those in a separate role/playbook for managing them, but I can't really provide support for that. Please do note that if you do extract them for use separately, you need to ensure you're respecting this project's license.

On another note, I think Collections are a way to get around this limitation? I haven't gotten a chance to study up on them so I'm not sure, but it's possible that what you're asking for might become more simple to do in the future. Not making any guarantees here though.

**Reply by mnarrell** [reply]
Thanks
